cfme/tests/cloud_infra_common/test_html5_vm_console.py

```diff
@@ -42,17 +27,39 @@ def vm_obj(request, provider, setup_provider, console_template):
                                     template_name=console_template.name)
 
     request.addfinalizer(lambda: vm_obj.cleanup_on_provider())
-
     vm_obj.create_on_provider(timeout=2400, find_in_cfme=True, allow_skip="default")
-    if provider.one_of(OpenStackProvider):
-        # Assign FloatingIP to Openstack Instance from pool
-        # so that we can SSH to it
-        public_net = provider.data['public_network']
-        vm_obj.mgmt.assign_floating_ip(public_net)
     return vm_obj
 
 
+@pytest.mark.tier(2)
+@test_requirements.html5
+@pytest.mark.provider([VMwareProvider], selector=ONE)
+@pytest.mark.usefixtures('setup_provider')
```

Comment:
Why are you marking for this test function, instead of including setup_appliance in its parameter list?

Reply:
I din't know about that fixture, I will update it accordingly to include `setup_appliance` in params list.

Reply:
Typo on my part - I meant setup_provider, but adding it to the test with usefixtures marker at a function level is strange. Its for use at the class/module/session level, where the fixture is added to all test nodes in the scope.